rn/Teacher/src/modules/graphql-speed-grader/components/SimilarityScore.js

```diff
@@ -0,0 +1,194 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2017-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+// @flow
+
+import React, { Component } from 'react'
+import i18n from 'format-message'
+import { createStyleSheet } from '../../../common/stylesheet'
+import images from '../../../images'
+import DisclosureIndicator from '../../../common/components/DisclosureIndicator'
+
+import {
+  Text,
+  TouchableHighlight,
+  View,
+  Image,
+} from 'react-native'
+
+type StateProps = {
+  status: ?string,
+  score: ?number,
+  url: ?string,
+}
+
+export type Props = StateProps
+
+const TURN_IT_IN_COLORS = {
+  PASS: '#008EE2',
+  DECENT: '#00AC18',
+  SKETCHY: '#FFC100',
+  DANGER: '#FC5E13',
+  THREAT_LEVEL_MIDNIGHT: '#EE0612',
```

Comment:
@vanwagonet removed these when he did dark mode. I think we probably want to keep with what he did. What do you think?

https://github.com/instructure/canvas-ios/blob/55979cec1be09cbe75e8884bc2df556b5cf96f3c/rn/Teacher/src/modules/speedgrader/components/SimilarityScore.js#L74-L91

Reply:
oh good to know. I will change that.